lib/game.rb

```diff
@@ -0,0 +1,172 @@
+class Game
+  attr_accessor :board,:player_1,:player_2
+
+WIN_COMBINATIONS_GAME = [
+  [0,1,2],
+  [3,4,5],
+  [6,7,8],
+  [0,3,6],
+  [1,4,7],
+  [2,5,8],
+  [0,4,8],
+  [6,4,2]
+  ]
+
+  def initialize(player_1 = Human.new("X"),player_2 = Human.new("O"),board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+  
+  def player_1
+    @player_1
```

Comment:
With `attr_accessor :player_1`, this method is not needed. `attr_accessor` is a macro that automatically adds both the setter method and the getter method to your class for the selected attribute. In this case, both
```
def player_1
  @player_1
end
```
and
```
def player_1=(player_1)
  @player_1 = player_1
end
```
have been included in your class through the `attr_accessor`.